ray_data_eval/image_training/sampling.py

```diff
@@ -1,18 +1,32 @@
 import os
```

Comment:
Rename this script to a better name (maybe `create_sample_dataset`) and add a one-liner docstring to describe what it does.

Reply:
Done, thanks!